cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -77,4 +77,9 @@ the label `cpumanager=true` is not automatically set on nodes. As a
 workaround, remove the `CPUManager` entry from the `kubevirt-config` ConfigMap.
 Then, manually label the nodes with `cpumanager=true` before running virtual
 machines with guaranteed CPUs on your cluster.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1718944[*BZ#1718944*])
\ No newline at end of file
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1718944[*BZ#1718944*])
+
+* If you use the web console to create a virtual machine template that has the same name as an existing
+virtual machine, the operation fails and the message `Name is already used by another virtual machine` is displayed.
+As a workaround, create the template from the command line.
```

Comment:
Do you want to provide a link to the assembly that describes how to do this? Otherwise it LGTM.

Reply:
I think this looks good, but I agree that an xref would be useful. I think it might be better to hold off on it for now, though, because the build failed when I added an xref to this file. I had to take it out before merging.